### PR TITLE
[ready] Fix extreme memory usages on /billables for accounts with 7000+ billable entries

### PR DESCRIPTION
**Pull request by dkhgh**
So we ran into an issue where the `/billables` view timed out when an account had over 7000 billable entries. 

There was a lot of overly complicated stuff going on so I removed most of that. 

Unfortunately I also had to remove the filters on the **Billables** view (for now). Hopefully we can bring them back soon.  
There's still a lot of room for performance enhancements here (easiest would be filters + pagination), but we'd like to review and merge this first, to prevent blocking issues for users with many entries.

### CODE REVIEW

File: app/views/billables/_hours_entries.html.haml [hunk 1]
```diff
@@ -8,14 +8,12 @@
...
     %th= t("billables.table.hours")
     %th= t("billables.table.user")
-  - entries.each do |entry|
+  - project.hours.where(billed: false).each do |hour|
     %tr.info-row
-      %td=billable_entry_checkbox(entry, "hours")
-      %td=I18n.l (entry.date)
-      %td=entry.category.name
-      %td.center= entry.value
+      %td=billable_entry_checkbox(hour, "hours")
```
Comment (dkhgh) [inline app/views/billables/_hours_entries.html.haml]: haml makes me sad 🌵

File: app/assets/stylesheets/_billables.scss [hunk 2]
```diff
@@ -26,10 +26,8 @@
 
     max-width: 90em;
 
-    .submit-billable-entries {
-      @include media($laptop-screen) {
-        opacity: 0;
-      }
+    #submit-billable-entries-test {
+      opacity: .0;
```
Comment (dkhgh) [inline app/assets/stylesheets/_billables.scss]: this is a hidden button to circumvent some issues with the Capybara webkit driver that doesn't play nice with our subdomains.

Was already there, just made it easier to find.

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -42,7 +42,7 @@ def current_locale
 
   def billable_entry_checkbox(entry, entry_type)
     if entry.billed
-      "√"
```
Comment (dkhgh) [inline app/helpers/application_helper.rb]: Seriously, right? 😂

**Comment by jurre** [comment]
Let's fix these rubocop violations before merging 👮 🐶

**Comment by fatboypunk** [comment]
All done now, ready to go?

**Comment by jurre** [comment]
Yes, I think so. I hope removing the filters here won't mess up anyone's workflow. But I guess we'll hear. Do we have an easy way for people who are using happyhours.io to contact us?

**Comment by fatboypunk** [comment]
I think we only have an helpful option in there and since they stopped i think there's no option to reach us anymore

**Comment by jurre** [comment]
Maybe link to github issues? Seems unrelated to this PR, but important though

**Comment by fatboypunk** [comment]
Let's solve these issues in other pull requests. Think we need this and the filters can still be used in the reporting.

File: app/helpers/application_helper.rb (hunk 3 above)
Comment (Marthyn) [inline app/helpers/application_helper.rb]: :joy: my bad

**Pull request merged**